Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js

```diff
@@ -81,6 +81,35 @@ export function getViewPlaneNameFromViewType(viewType) {
   }
 }
 
+// Get the names and normals of other views
+// currentNormal = normal of the current view
+// It will return an object that contains
+export function getOtherViews(currentNormal, widgetState) {
+  const XPlaneNormal = widgetState.getXPlaneNormal();
+  const YPlaneNormal = widgetState.getYPlaneNormal();
+  const ZPlaneNormal = widgetState.getZPlaneNormal();
+
+  const viewX = {
+    name: 'X',
+    normal: XPlaneNormal,
+  };
+  const viewY = {
+    name: 'Y',
+    normal: YPlaneNormal,
+  };
+  const viewZ = {
+    name: 'Z',
+    normal: ZPlaneNormal,
+  };
+  let otherViews = [viewX, viewY];
+  if (vtkMath.dot(currentNormal, XPlaneNormal) > 0.9) {
```

Comment:
this does not seem very robust (ViewX and ViewY can eventually share the same normal).
You should probably use `viewType` to find other views.

Reply:
`viewType `? The main issue I have here is that I cannot know in which view I am. I only catch the mouse wheel on a renderer but I don't know which one of the three.

Reply:
Isn't `model.widgetState.getActiveViewName()` returning the current viewType ?

Reply:
Unfortunately not. These state attribute is only set when you select a pickable object:
https://github.com/Kitware/vtk-js/blob/master/Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js#L184

Reply:
In behavior.js , you can do `model.viewType`

Reply:
Indeed ! My bad. It's really such a better way to do that. Thanks!